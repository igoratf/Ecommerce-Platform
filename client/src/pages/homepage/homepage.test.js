import { shallow } from 'enzyme';
import React from 'react';
import Homepage from './homepage.component';

it('expect to render home page', () => {
  expect(shallow(<Homepage />)).toMatchSnapshot();
})
