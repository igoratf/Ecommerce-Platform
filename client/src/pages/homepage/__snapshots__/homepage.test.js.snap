// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render home page 1`] = `ShallowWrapper {}`;
